src/m_argv.c

```diff
@@ -153,6 +158,11 @@ static void LoadResponseFile(int argv_index, const char *filename)
             break;
         }
 
+        if (newargc == MAXARGVS)
```

Comment:
This looks out of place. It should be immediately before or after changing `newargc`.